design/FY2022/NFP-RefCondsWtrToAirCoils.md

```diff
@@ -0,0 +1,283 @@
+
+The total cooling and heating coils capacity is currently solely based on cooling peak calculations. It is possible that in certain climate zones, and certain buildings, that heating load dominates cooling loads and thus that heating coils might end-up being undersized. We proposed to add a heating peak calculation to calculate the heating load that the coil would have to meet, and using the the ratio of heating reference capacity to reference cooling capacity calculate the corresponding reference cooling coil capacity and set the cooling capacity to be the maximum between this one and the one determined based on cooling peak calculations.
+
+### Example ###
+
+The following example illustrates the issue and proposed approach. Let's consider a model using a `Coil:Cooling:WaterToAirHeatPump:EquationFit` using the same performance curves as in the `ASHRAE901_ApartmentHighRise_STD2019_Denver.idf` file.
+
+Let's assume that:
+- The coil load at peak design conditions is `CoolCapAtPeak` = 1000W, 
+- The entering air wet-bulb temperature at peak design conditions is `MixWetBulb` = 15.6&deg;C, 
+- The entering air dry-bulb temperature at peak design conditions is `MixTemp` = 21.1&deg;C, 
+- The user-specified cooling COP at rated conditions is `RatedCOPCool` = 4.2
+
+#### Current Approach ###
+_Note: The current variable names are used below._
+
+- `PeakTotCapTempModFac` is calculated at 29.44&deg;C (85&deg;F) EWT and 15.6&deg;C (60&deg;F) EAWT and is about 0.873
+- `RatedCapCoolTotalDes` = 1000 / 0.873 = 1145 W
+- `RatedPowerCool` = 1145 / 4.2 = 273 W
+
+The rating conditions for water-to-air water loop heat pumps are 30&deg;C (86&deg;F) EWT and 19&deg;C (66.2&deg;F), at these conditions the `TotCapTempModFac` is 0.956 and `PowerTempModFac` is 1.006. When using these modifiers to calculate the operating capacity and power at these conditions we get a capacity of 1145 * 0.956 ~= 1095 W and a power of 274 * 1.006 ~= 274 W which correspond to a COP of 1095 / 274 ~= 4 which doesn't align with the user-specified COP of 4.2.
+
```

Comment:
Where did these curves come from? Each curve should have a value of 1 at the rating point.

Reply:
What is the PowerTempModFac at the design condition of 15.6 C? It seems that value should be used to calculate RatedPowerCool.

Reply:
Is seems this math doesn't work because the CapFt at the rating point is not 1.0.

1000 / (0.873 * 0.956) = 1198.2

I am not sure here if PowerTempModFac = 1 at this design point (15.6C EWT) but 1198.2 / (4.2 * 1/ (1.0 * 1.006)) = 286.99. Then 1198.2/(286.99*1.006) = 4.2. If PowerTempModFac is not 1 at this design point, and is not 1 at the rating point, which you say is 1.006, then these multipliers should be included in the rated power calculation. It seems wierd to be multiplying by fractions of capacity and power at the rating point. I used 1/PowerTempModFac because COP is capacity/power.

Reply:
For example file HVACTemplate-5ZoneUnitarySystem, how does the autosized capacity and power compare to the cooling COP = 3 and heating COP = 2.75?